lineman/spec/models/discussion_model_spec.coffee

```diff
@@ -61,3 +62,30 @@ describe 'DiscussionModel', ->
   describe 'reader', ->
     it "returns the discussion reader associated with this discussion", ->
       expect(discussion.reader()).toBe(discussionReader)
+
+  describe 'clone()', ->
+    beforeEach ->
+      window.Loomio =
+        permittedParams:
+          discussion: ['title', 'author_id']
+
+    it 'copies all of the attributes of the object being cloned', ->
+      expect(discussion.clone().title).toBe(discussion.title)
+
+    it 'copies all of the methods of the object being closed', ->
+      expect(discussion.clone().authorName()).toBe(discussion.authorName())
+
+    it 'does not create a new record', ->
```

Comment:
I have to note that a couple of these are failing at the moment, because we aren't currently simulating the bit where we post to the server and the updating values get imported back into the record store.